packages/http/src/types.ts

```diff
@@ -17,6 +17,11 @@ export interface IHttpConfig extends IPrismConfig {
   mock: false | IHttpOperationConfig;
 }
 
+export interface IHttpProxyConfig extends IHttpConfig {
```

Comment:
I think there is sth not right with the config types.

Below looks more reasonable:
`IHttpProxyConfig extends IHttpBaseConfig { upstream: .. }`
`IHttpMockConfig extends IHttpBaseConfig { mock: .. }`
`IHttpConfig { common stuff }`

WDYT?

Reply:
Isn't is in this way already?

```ts
export interface IHttpConfig extends IPrismConfig {
  mock: false | IHttpOperationConfig;
}

export interface IHttpProxyConfig extends IHttpConfig {
  mock: false;
  upstream: URL;
}

export type IMockHttpConfig = IHttpConfig & { mock: IHttpOperationConfig };

```

Reply:
5ecf360ca253ce09a6668e2c107e02ef5862133f